test/plugin/scenarios/gson-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+  applications:
+    - {gson-scenario: nq 0}
+  instances:
+    - {gson-scenario: nq 0}
+  operationNames:
+    - gson-scenario: [/gson-scenario/case/healthCheck,/gson-scenario/case/gson-scenario]
```

Comment:
healthCheck should not be included in expected data today.

Reply:
I will remove.